### PR TITLE
Updated KeywordManager to support key bindings as well

### PR DESCRIPTION
**Pull request by alexdrenea**
This allows quick and easy debugging in the Unity Editor
Fixes Issue #36.

### CODE REVIEW

File: Assets/HoloToolkit/Input/Scripts/KeywordManager.cs [hunk 1]
```diff
@@ -67,6 +69,18 @@ void Start()
             }
         }
 
+        void Update()
+        {
+            foreach (var kvp in KeywordsAndResponses)
```
Comment (NeerajW) [inline Assets/HoloToolkit/Input/Scripts/KeywordManager.cs]: Like the idea, but please put this into its own function for reusability.

**Comment by alexdrenea** [comment]
Good point. Thanks Neeraj.

**Comment by jwittner** [comment]
:+1:

File: Assets/HoloToolkit/Input/Scripts/KeywordManager.cs [hunk 2]
```diff
@@ -77,6 +84,18 @@ void OnDestroy()
             }
         }
 
+        private void ProcessKeyBindings()
+        {
+            foreach (var kvp in KeywordsAndResponses)
+            {
+                if (Input.GetKeyDown(kvp.KeyCode))
```
Comment (angelaHillier) [inline Assets/HoloToolkit/Input/Scripts/KeywordManager.cs]: You should change this to GetKeyUp (so the call happens on release of the keyboard key).

**Comment by angelaHillier** [comment]
Simple enough change. Merging with main.

**Pull request merged**